Il2CppInterop.Runtime/Injection/ClassInjector.cs

```diff
@@ -847,10 +864,9 @@ private static Delegate CreateInvoker(MethodInfo monoMethod)
                 body.Emit(OpCodes.Ldobj, nativeType);
         }
 
-        body.Emit(OpCodes.Ldarg_0);
-        body.EmitCalli(OpCodes.Calli, CallingConvention.Cdecl, monoMethod.ReturnType.NativeType(),
-            new[] { typeof(IntPtr) }.Concat(monoMethod.GetParameters().Select(it => it.ParameterType.NativeType()))
-                .ToArray());
+        body.Emit(OpCodes.Ldarg_1); // methodMetadata
```

Comment:
On the surface, this looks like it will break instance methods. However, I saw this elsewhere:
```cs
nativeParameterTypes.Add(typeof(Il2CppMethodInfo*));
```
so I think this line is actually fine.